index.bs

```diff
@@ -2128,19 +2127,19 @@ An {{AudioBuffer}} may be used by one or more
 {{AudioBuffer}} has four internal slots:
 
 <dl dfn-type=attribute dfn-for="AudioBuffer">
-	: <dfn>\[[number of channels]]</dfn>
+	: <dfn attribute for="AudioBuffer">[[number of channels]]</dfn>
```

Comment:
The "attribute" and "for" are already taken care of by the parent element, all you need here (and in the following spots) are `: <dfn>[[number of channels]]</dfn>`.

Reply:
Done.